app/experimenter/visualization/api/v1/views.py

```diff
@@ -0,0 +1,38 @@
+import json
+
+from django.core.files.storage import default_storage
+from django.http import HttpResponse
+from django.shortcuts import get_object_or_404
+from rest_framework.decorators import api_view
+
+from experimenter.experiments.models import NimbusExperiment
+
+
+def load_data_from_gcs(filename):
+    return (
+        json.loads(default_storage.open(filename).read())
+        if default_storage.exists(filename)
+        else None
+    )
+
+
+@api_view()
+def analysis_results_view(request, slug):
+    experiment = get_object_or_404(NimbusExperiment.objects.filter(slug=slug))
+    experiment_json = {}
```

Comment:
Nit: this is getting supernitpicky but I usually I use the suffixes `_data` if it's a dict that came from some json and `_json` if it's the actual json string.  But that's just me.